src/main/java/net/rptools/maptool/client/ui/commandpanel/CommandPanel.java

```diff
@@ -441,7 +441,7 @@ public void keyReleased(KeyEvent kre) {
    * FIXME: this is insufficient for stopping faked rolls; the user can still do something like &{"laquo;"}.
    */
   public static final Pattern CHEATER_PATTERN =
-      Pattern.compile("\u00AB|\u00BB|&#171;|&#187;|&laquo;|&raquo;|\036|\037");
+      Pattern.compile("\u00AB|\u00BB|&#171|&#187|&laquo|&raquo|\036|\037|&#xAB|&#xBB");
```

Comment:
Could you please re-add the semicolons, but add a question mark after them so they become optional?
```suggestion
      Pattern.compile("\u00AB|\u00BB|&#171;?|&#187;?|&laquo;?|&raquo;?|\036|\037|&#xAB|&#xBB");
```

Reply:
Sure. I also added the ;? to &#xAB and &#xBB.